omnibridge/flows/graph_flow.py

```diff
@@ -0,0 +1,73 @@
+from typing import List, Dict, Any, Union
+
+from omnibridge.model_entities.models_io.base_model_io import ModelIO, TextualIO
+from omnibridge.model_entities.models_io.flow_io import FlowIO
+from omnibridge.saved_data.json_data_manager import JsonConvertable
+from omnibridge.wrappers.wrapper_instances.type_name_to_wrapper import ModelLoader
+from omnibridge.wrappers.wrapper_interfaces.model_wrapper import ModelWrapper
+
+
+class Node(JsonConvertable):
+    def __init__(self, model: ModelWrapper, instruction: str, next_nodes: List[ModelWrapper]):
+        self.next_nodes = next_nodes
+        self.model = model
+        self.instruction = instruction
+
+    def process(self, model_input: ModelIO) -> ModelIO:
+        assert isinstance(model_input, TextualIO)
+        model_input = TextualIO(model_input.get_text() + "\n" + self.instruction)
+        model_output = self.model.process(model_input)
+        aggregate_output = FlowIO(model_output)
+        for next_model in self.next_nodes:
+            next_model_output = next_model.process(model_output)
+            aggregate_output += next_model_output
+
+        return aggregate_output
+
+    def to_json(self) -> Dict[str, Union[str, int]]:
+        raise NotImplementedError()
+
+    @classmethod
+    def create_from_json(cls, json_key: str, json_data: Dict[str, str]) -> Any:
+        model_name = json_data['name']
+        model = ModelLoader.load_model(model_name)
+
+        instruction = ""
+        if 'instruction' in json_data:
+            instruction = json_data['instruction']
```

Comment:
just a suggestion, IMO it is clearer that way

```suggestion
        instruction = json_data.get('instruction', '')
```